client/packages/coldchain/src/Monitoring/api/TemperatureBreach/hooks/document/useUpdateTemperatureBreach.ts

```diff
@@ -0,0 +1,16 @@
+import { useTemperatureNotificationApi } from '../../../TemperatureNotification/hooks/utils/useTemperatureNotificationApi';
+import { useTemperatureBreachApi } from '../utils/useTemperatureBreachesApi';
+import { useMutation, useQueryClient } from '@openmsupply-client/common';
+
+export const useUpdateTemperatureBreach = () => {
+  const queryClient = useQueryClient();
+  const api = useTemperatureBreachApi();
+  const notificationApi = useTemperatureNotificationApi();
+
+  return useMutation(api.update, {
+    onSuccess: () => {
+      queryClient.invalidateQueries(api.keys.base());
+      queryClient.invalidateQueries(notificationApi.keys.base());
```

Comment:
they differ, in that the breach query has filter/sort/paginate and the notification is a simpler api, which supports some aspects but is intended to show a single breach in the notification area only.
Another way to simplify is to change the notification cache key so that it depends on the base breach key though - that way if the breaches are invalidated, the notification refetches. that's neat, I think